spec.html

```diff
@@ -111,10 +111,10 @@ <h1>
 			1. Let _callerContext_ be the running execution context.
 			1. Let _calleeContext_ be PrepareForWrappedFunctionCall(_F_).
 			1. Assert: _calleeContext_ is now the running execution context.
-			1. Let result be OrdinaryWrappedFunctionCall(_F_, _thisArgument_, _argumentsList_).
+			1. Let _result_ be Completion(OrdinaryWrappedFunctionCall(_F_, _thisArgument_, _argumentsList_)).
```

Comment:
Since `OrdinaryWrappedFunctionCall` can throw... at multiple places... better to be sure here.